### ISSUE TITLE
Add BotMate

### ISSUE THREAD

**Issue by mxvsh** [awesome-selfhosted/awesome-selfhosted-data#874]
Thanks for taking the time to suggest an addition to awesome-selfhosted!

Please fill out information below (all fields are mandatory unless noted otherwise):

```yaml
name: 'BotMate'
website_url: 'https://botmate.dev'
source_code_url: 'https://github.com/botmate/botmate'
description: 'BotMate is a platform that allows users to manage multiple bots across platforms like Telegram and Discord from one place. It simplifies bot management through a customizable dashboard and supports plugin-based functionality. With easy-to-build plugins using modern technologies like TypeScript, React, and Node.js, developers can extend bots' capabilities effortlessly. BotMate offers a marketplace for plugins, enabling users to quickly install or remove features in just one click. It’s ideal for both developers and non-technical users who need a simple way to manage bots.'

licenses:
  - MIT
platforms:
  - React
  - TypeScript
  - Nodejs
  - TailwindCSS
  - MongoDB
  - Docker
tags:
  - Automation
  - Software Development
  - Software Development - Low Code
related_software_url: 'https://docs.botmate.dev'
```

To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):

- [x] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.

**Comment by Rabenherz112** [comment]
Hi, thanks for your contribution.

Currently, this project has a release, but it is not yet 4 months old. Our guidelines require that _Any software project you are adding was first released more than 4 months ago_. This count initiates only after a release has been created to ensure users need not rely on the latest development version to use the project.

The first release was on Jun 30, so let's wait until `2025-01-25`, when your release is four months old.

**Comment by Ki-er** [comment]
No commits in 5 months, lets close this for now to prevent stale issues stuck in needs to mature limbo.

We can reopen this or submit new issue if this project is reactive.
